mw-webapp/src/styles/_variables.scss

```diff
@@ -10,6 +10,7 @@ $link-active: rgb(255, 149, 0);
 
 // Header
 $header-background: rgb(85, 122, 149);
+$header-padding: 0 1em;
```

Comment:
What do zou think about using rem instead of em?

Reply:
I agree, rem is more global